Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM phusion/baseimage:0.9.16
+FROM phusion/baseimage
```

Comment:
Probably not needed anymore.  There was an issue with the docker repo when I was testing this and pulling down that version, as well as some ca-cert issues, but you can probably tack it to the latest if you wanted to.